modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -180,6 +181,43 @@ bool pyopencv_to(PyObject* obj, cv::gapi::wip::draw::Prim& value, const ArgInfo&
     return false;
 }
 
+template <>
+bool pyopencv_to(PyObject* obj, cv::GMetaArg& value, const ArgInfo&)
+{
+    if (PyObject_TypeCheck(obj,
+                reinterpret_cast<PyTypeObject*>(pyopencv_GMatDesc_TypePtr)))
+    {
+        value = reinterpret_cast<pyopencv_GMatDesc_t*>(obj)->v;
+    }
+    else if (PyObject_TypeCheck(obj,
+                reinterpret_cast<PyTypeObject*>(pyopencv_GScalarDesc_TypePtr)))
+    {
+        value = reinterpret_cast<pyopencv_GScalarDesc_t*>(obj)->v;
+    }
+    else if (PyObject_TypeCheck(obj,
+                reinterpret_cast<PyTypeObject*>(pyopencv_GArrayDesc_TypePtr)))
+    {
+        value = reinterpret_cast<pyopencv_GArrayDesc_t*>(obj)->v;
+    }
+    else if (PyObject_TypeCheck(obj,
+                reinterpret_cast<PyTypeObject*>(pyopencv_GOpaqueDesc_TypePtr)))
+    {
+        value = reinterpret_cast<pyopencv_GOpaqueDesc_t*>(obj)->v;
+    }
+    else
+    {
+        failmsg("Unsupported cv::GMetaArg type");
+        return false;
+    }
+    return true;
+}
+
+template <>
+bool pyopencv_to(PyObject* obj, cv::GMetaArgs& value, const ArgInfo& info)
+{
+    return pyopencv_to_generic_vec(obj, value, info);
+}
```

Comment:
I suggest to put all of it after `pyopencv_to(obj, cv::gapi::wip::draw::Prims&)` (the next overload) not to separate `Prim` and `Prims` overloads

Reply:
Done

Reply:
Done